README.md

```diff
@@ -27,8 +27,10 @@ You can get started by creating an account on Azure. There's three different opt
 
 ## Student Resources
 
-- [Azure Student Labs](https://aka.ms/azurestudentlabs) - Our repository of labs geared for computer science students that you can learn from. Find them at [aka.ms/azurestudentlabs](https://aka.ms/azurestudentlabs)
+- [Learning paths for students](https://aka.ms/student-learn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [aka.ms/student-learn](https://aka.ms/student-learn)

```

Comment:
Can we remove the callout to aka.ms itself as it's not accessible? Also, a couple of sentences on what the learn content looks like/is?

Thanks!

Reply:
```suggestion
[Learning paths for students](https://aka.ms/StudentsLearn) - Hands-on learning paths and modules designed for students. Find them on Microsoft Learn at [Learning paths for students - aka.ms/StudentsLearn](https://aka.ms/StudentsLearn)

```